PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.cxx

```diff
@@ -2493,12 +2532,24 @@ AliEmcalJet* AliAnalysisTaskEmcalJetPerformance::GetMatchedJet(AliEmcalJet* detJ
 
   AliEmcalJet* matchedTruthJetCand = detJet->ClosestJet();
   
+  // Check that matched truth jet exists and is a unique match
   if (!matchedTruthJetCand) {
     return 0;
   }
   if (matchedTruthJetCand->ClosestJet() != detJet) {
     return 0;
   }
+  
+  // Fill matching distance between combined jet and pp det-level jet, before imposing deltaR cut (centrality, pT-truth, R)
+  TString histname = "MatchedJetHistograms/hMatchingDistance";
+  if (fForceBeamType != kpp) {
+    fHistManager.FillTH3(histname, fCent, matchedTruthJetCand->Pt(), detJet->ClosestJetDistance());
```

Comment:
Be really careful with this. This is the kind of thing that I could imagine would cause the TH3 fill error that you saw in embedding

Reply:
Sure but I thought we both agreed that the tagger problem can't possibly have anything to do with my task? I will keep an eye for the error showing up elsewhere, but I don't see any reason to suspect FillTH3 misbehaves, I have been using it extensively for a long time.

Reply:
Yeah, you're right, my bad - it struck me at first glance, but you're right that it shouldn't matter

Reply:
Ok, cool -- thanks for the review :)